references/detection/transforms.py

```diff
@@ -52,6 +52,20 @@ def forward(self, image: Tensor,
         return image, target
 
 
+class PILToTensor(nn.Module):
+    def forward(self, image: Tensor,
+                target: Optional[Dict[str, Tensor]] = None) -> Tuple[Tensor, Optional[Dict[str, Tensor]]]:
+        image = F.pil_to_tensor(image)
+        return image, target
+
+
+class ConvertImageDtype(nn.Module):
```

Comment:
How about something like this?
```python
    def __init__(self, dtype: torch.dtype) -> None:
        super().__init__()
        self.dtype = dtype
```

Reply:
Makes sense. Should we have a default value for dtype(torch.float)?

Reply:
I was proposing to completely align it with the actual `ConvertImageDtype` of TorchVision. I believe it doesn't set one. I think aligning them will make it super useful to switch to the new Transformations API that Francisco and Phillip are working on. That's the idea at least. Let me know if you agree.

Reply:
Okay, sounds good.